ui/components/NFTS_update/Filters/NFTsFilters.tsx

```diff
@@ -153,13 +154,18 @@ export default function NFTsFilters(): ReactElement {
           gap: 16px;
           height: 456px;
           overflow-y: scroll;
-          padding: 8px 24px;
+          padding: 0 24px 8px;
         }
         .filter_title {
           display: inline-block;
           margin-bottom: 4px;
           width: 100%;
         }
+        .filter_warning {
+          font-size: 16px;
+          line-height: 24px;
+          color: var(--green-20);
```

Comment:
These properties are already in the `simple_text`  class. I think in this class we can only set the color.
```suggestion
          color: var(--green-20);
```